streaming/multimodal/convert/webvid.py

```diff
@@ -0,0 +1,158 @@
+# Copyright 2023 MosaicML Streaming authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Download videos, creating an MDS dataset.
+
+Instructions:
+
+1. Navigate to the download section of https://m-bain.github.io/webvid-dataset/, where you will
+   find 2.5M and 10M dataset splits:
```

Comment:
is it 2.5M or 2M?

Reply:
hmm. googling around, i see both terms being used. it is 2.5m samples